src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/payment/processor/client/payhub/PayHubClientTest.java

```diff
@@ -28,29 +27,27 @@
 import static org.springframework.http.HttpStatus.BAD_REQUEST;
 import static org.springframework.http.HttpStatus.CONFLICT;
 import static org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR;
+import static uk.gov.hmcts.reform.bulkscan.payment.processor.util.TestUtil.fileContentAsString;
 
 @IntegrationTest
 public class PayHubClientTest {
 
     @Autowired
     private PayHubClient client;
 
-    @Autowired
-    private ObjectMapper mapper;
-
     @Test
     public void should_return_Ok_when_everything_is_ok_with_request() throws JsonProcessingException {
         // given
         String s2sToken = randomUUID().toString();
 
-        PaymentResult response = new PaymentResult(ImmutableList.of("xxxyyyzzz", "zzzyyyxxx"));
-
         stubWithRequestAndResponse(
             s2sToken,
-            mapper.writeValueAsString(getPaymentRequest()),
-            okJson(mapper.writeValueAsString(response))
+            fileContentAsString("testdata/bulk-scan-payments/payment-request.json"),
```

Comment:
it would be nice to keep the dcns in the request json file same as the response

Reply:
done